src/handlers/ens.ts

```diff
@@ -861,10 +907,9 @@ export const shouldUseMulticallTransaction = (
     text,
   } = registrationRecords;
   if (
-    !coinAddress?.length &&
     !contentHash &&
     !ensAssociatedAddress &&
-    text?.length === 1
+    (text?.length || 0) + (coinAddress?.length || 0) === 1
```

Comment:
ah ok i see this condition to not use muticall when there's only one text or only one coinAddress